homeassistant/components/light/osramlightify.py

```diff
@@ -19,7 +19,8 @@
     SUPPORT_COLOR_TEMP, SUPPORT_RGB_COLOR, SUPPORT_TRANSITION, PLATFORM_SCHEMA)
 import homeassistant.helpers.config_validation as cv
 
-REQUIREMENTS = ['lightify==1.0.3']
+REQUIREMENTS = ['https://github.com/tfriedel/python-lightify/archive/'
+                'master.zip#lightify==0.0.4']
```

Comment:
Please don't point it at the master branch but at the zip of a specific commit.

You also need to specify the exact version number as is contained in the zip, so I think 1.0.4 ?